engine/src/flutter/lib/ui/text.dart

```diff
@@ -53,42 +55,47 @@ enum FontStyle {
 /// with the name "Roboto" and the weight [FontWeight.w500].
 ///
 /// Some modern fonts allow the weight to be adjusted in arbitrary increments.
-/// See [FontVariation.weight] for details.
+/// When using these fonts, applications can specify [FontWeight] instances
+/// constructed using values other than the predefined values.
 class FontWeight {
```

Comment:
I wonder if this can be an extension type now that it is just an `int` under the hood?

Reply:
Ah there are FontWeight subclasses in the wild but since we were using `index` to communicate the weight I don't think subclassing does anything.